src/extra/calibration.py

```diff
@@ -35,6 +39,14 @@
 CALCAT_PROXY_URL = "http://exflcalproxy.desy.de:8080/"
 
 
+def any_is_none(*values):
```

Comment:
Not remotely important to the overall PR, but this can be one line with the builtin [`any()`](https://docs.python.org/3/library/functions.html#any) function.